README.md

```diff
@@ -10,6 +10,15 @@ CZ CELLxGENE Explorer (pronounced "cell-by-gene") is an interactive data explore
 
 Whether you need to visualize one thousand cells or one million, CELLxGENE Explorer helps you gain insight into your single-cell data.
 
+#### Highlighted Features
+
+- **Interactive Exploration**: Provides intuitive navigation of 2D matrices with dimensions, allowing users to explore and annotate datasets.
+- **Data Visualization**: Offers clear, intuitive visual representations of datasets and related artifacts.
+- **Spatial Mode with Deep Zoom**: Enables seamless zooming in and out of high-resolution images & spatial data (e.g., Slide-seq V2, Visium) to examine fine details.
+- **Gating/Selection**: Allows users to sub-select data and integrate multiple selections.
+- **Annotation**: Users can label cells and create gene sets.
```

Comment:
Hmm we can indeed create gene sets but not labeling cells, so maybe change `Annotation` to `Customization` or something better 💡 ?